src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -115,15 +115,15 @@ trait Formats
     val base = Json.obj(
       "id" -> task.taskId,
       "slaveId" -> task.agentInfo.agentId,
-      "host" -> task.agentInfo.host
+      "host" -> task.agentInfo.host,
+      "ipAddresses" -> task.ipAddresses
```

Comment:
So it is important to you to have an empty list there disregarding the task state instead of not having the ipAddresses field in case the task is not "launched"?